docs/pages/access-controls/guides/impersonation.mdx

```diff
@@ -5,12 +5,13 @@ description: How to issue short-lived certs on behalf of Teleport users using im
 
 # Impersonation
 
-Sometimes users need to create short-lived certificates for non-interactive users, for example, CI/CD systems.
-Your programs interacting with Teleport may need to create their own authentication as well.
-Teleport's impersonation allows users and robots to create short-lived certs for other users and roles.
+Sometimes users need to create short-lived certificates for non-interactive
```

Comment:
Thanks for the PR! As you started contributing to the docs, check out our `meta` guide on style here:

https://goteleport.com/docs/docs/